README.md

```diff
@@ -101,6 +101,29 @@ await page.RouteAsync("**", (route, _) =>
 await page.GotoAsync("http://todomvc.com");
 ```
 
+## CLI & Codegen
+
+Playwright conveniently comes with a built-in [`codegen` facility](https://playwright.dev/dotnet/docs/cli) which lets you open an inspector and a browser window, then record the 
+actions you perform, generating the relevant source code as you do.
+
+To use it, you need to install the dotnet tool:
+
+```powershell
+dotnet-install --global microsoft.playwright.cli
+```
+
+Then, you can simply call:
+
+```powershell
+playwrightcli codegen https://www.microsoft.com
+```
+
+### Driver Discoverability
+
+By default, the tool will attempt to look at `%USERPROFILE%\.nuget\packages\microsoft.playwright` for the location of the driver. If that behaviour
+is undesired, you can set the `PW_CLI_DRIVERPATH` environment variable, to specify the location. The location needs to be *specific* and point to the
```

Comment:
I don't think anyone should set this variable that points to internal Playwright file.